src/vm/comdelegate.cpp

```diff
@@ -3849,17 +3851,22 @@ static void InvokeNotify(OBJECTREF *pDelegate, OBJECTREF *pDomain)
     }
     EX_CATCH
     {
+#ifndef FEATURE_CORECLR
```

Comment:
We often have a dummy implementations that just return false when the feature is stripped, to reduce number of ifdefs.